src/app/results/results.component.ts

```diff
@@ -28,7 +29,7 @@ export class ResultsComponent implements OnInit {
   query: any;
   count: number = 1;
   boxMessage = 'Show';
-
+  newsResponse: Array<any>;
```

Comment:
Don't we know the response of news?

Reply:
It is used to collect news from all the sources and store it locally.For each page it collects approx 10 news links.

Reply:
what is the type of response?

Reply:
It is Array.

Reply:
What is the type of value array is going to store?